packages/grafana-e2e/test/cypress/tsconfig.json

```diff
@@ -1,5 +1,4 @@
 {
-  "extends": "@grafana/toolkit/src/config/tsconfig.plugin.json",
```

Comment:
This change can potentially break typescript for the grafana-e2e package since you are probably removing a big part of the typescript configuration.

Reply:
Am I missing something or did we accidentally remove this tsconfig file from toolkit too early? https://www.unpkg.com/browse/@grafana/toolkit@10.0.4/src/ 

Whatever was there should probably be copied over into this file?

Reply:
We have discussed this with Esteban and it seems, this is unused, but I changed it to use  "@grafana/tsconfig" to be on safe side